src/backends/plotlyjs.jl

```diff
@@ -42,8 +42,8 @@ _display(plt::Plot{PlotlyJSBackend}) = display(plotlyjs_syncplot(plt))
 
 @require WebIO = "0f1e0344-ec1d-5b48-a673-e5cf874b6c29" begin
     function WebIO.render(plt::Plot{PlotlyJSBackend})
-        prepare_output(plt)
-        WebIO.render(plt.o)
+        plt_html = sprint(show, MIME("text/html"), x)
```

Comment:
I have not tested this, but isn't `x` undefined here?

Reply:
This is what I get for trying to write PR's using GitHub's web interface. You're correct.